sw/device/lib/dif/dif_clkmgr_unittest.cc

```diff
@@ -22,6 +22,57 @@ class ClkMgrTest : public Test, public MmioTest {
   dif_clkmgr_t clkmgr_ = {.base_addr = dev().region()};
 };
 
+class JitterEnableTest : public ClkMgrTest {};
+
+// SetEnabled uses EXPECT_WRITE32 instead of EXPECT_MASK32 because
+// dif_clkmgr_jitter_set_enabled doesn't perform a read, just a write.
+TEST_F(JitterEnableTest, SetEnabled) {
+  // Disable jitter.
+  EXPECT_WRITE32(CLKMGR_JITTER_ENABLE_REG_OFFSET, 0);
+  EXPECT_EQ(dif_clkmgr_jitter_set_enabled(&clkmgr_, kDifToggleDisabled),
+            kDifOk);
+
+  // Enable jitter.
+  EXPECT_WRITE32(CLKMGR_JITTER_ENABLE_REG_OFFSET, 1);
+  EXPECT_EQ(dif_clkmgr_jitter_set_enabled(&clkmgr_, kDifToggleEnabled), kDifOk);
+}
+
+TEST_F(JitterEnableTest, SetEnabledError) {
+  // Null handle.
+  EXPECT_EQ(dif_clkmgr_jitter_set_enabled(nullptr, kDifToggleEnabled),
+            kDifBadArg);
+}
+
+TEST_F(JitterEnableTest, GetEnabled) {
+  // Get jitter (enabled).
+  {
+    bool enabled = false;
```

Comment:
Can you split these two tests up into separate unit tests?

Reply:
I would prefer not to, since I see a good number of other corresponding tests set of get both values in a single test.